src/main/ngapp/accio/src/app/editor/editor.component.ts

```diff
@@ -400,7 +400,7 @@ export class EditorComponent implements OnInit {
   }
 
   /**
-   *  Makes a png of mask so the the background is transparent.
+   *  Makes a pewtopg of mask so the the background is transparent.
```

Comment:
Is this a typo?

Reply:
I think I accidentally hit keys without knowing it oops